Consul.Test/OperatorTest.cs

```diff
@@ -123,5 +123,17 @@ public async Task Operator_AreaList()
             Assert.Equal(area.UseTLS, result.UseTLS);
             Assert.Equal(area.RetryJoin, result.RetryJoin);
         }
+        [EnterpriseOnlyFact]
+        public async Task Operator_AreaUpdate()
+        {
+            var peerDataCenter = KVTest.GenerateTestKeyName();
+            var area = new AreaRequest { PeerDatacenter = peerDataCenter, UseTLS = false, RetryJoin = null };
+            var createResponse = await _client.Operator.AreaCreate(area);
+
+            area = new AreaRequest { PeerDatacenter = peerDataCenter, UseTLS = true, RetryJoin = new string[] { "10.1.2.9", "10.1.2.0" } };
+            var updateResponse = await _client.Operator.AreaUpdate(area, createResponse.Response);
+
+            Assert.NotNull(updateResponse.Response);
```

Comment:
Can we check what is exactly the content in the response?
Also it would be great to check if the Area properties,  before and after the update, are different.